### PR TITLE
Enabling the latest Graphviz attributes & shapes

### LINKED ISSUE #11: Newer attributes not considered valid
Hi folks.

When trying to use the `class` attribute for a node, we receive an error that it is not a valid attribute in the 'node' context.

The attribute is valid according to Graphviz - https://graphviz.org/docs/attrs/class/

I've poked around in the GraphViz2 commit history and found a commit (f254bacd3) where 5 new items (including `class`) were added to the `data/attributes.dat` file, but they have not yet been added in the `lib/GraphViz2.pm` file (and so they are not treated as valid).

I _think_ all that's needed to add support for these new attributes is to add the same new lines to `lib/GraphViz2.pm`? (I could be wrong.)

Below is the list of lines which are present in `data/attributes.dat` and not yet added to the [`@@ common_attribute` section of GraphViz2.pm](https://github.com/graphviz-perl/GraphViz2/blob/master/lib/GraphViz2.pm#L1863):

```
_background => graph
class => edge, node, cluster, graph
imagepos => node
newrank => graph
notranslate => graph
```
Could these newer attributes be added?
If it would help, I can put together a pull request adding these lines to the module file.

Thanks for your work on this module.

### PR DESCRIPTION
**Pull request by nowthis**
Resolves graphviz-perl/GraphViz2#11.

High-level summary of the 3 commits:
* The scripts reading values from graphviz.org pages are updated to adapt to that site's current format.
* All new known shapes and attributes are enabled in `GraphViz2.pm`.
* A test exercising the `class` attribute is added.
  * Inside the test script are comments documenting steps walking through how to see some sample CSS effects on the test's output.
  * Documentation of the results of those steps is posted at https://github.com/nowthis/GraphViz2/commit/1f25b684e1694eca782d82470f6bf131a5a9935f#comments

### CODE REVIEW

File: scripts/extract.arrow.shapes.pl [hunk 1]
```diff
@@ -11,7 +11,7 @@
 my(@node)      = $root -> look_down(_tag => 'table');
 my @td         = $node[1]->look_down(_tag => 'td');
 
-my @shape = map /"(.+)"/ ? $1 : (), map +($_->content_list)[0], @td;
+my @shape = grep { length $_ } map { $_->as_text() } @td;
```
Comment (mohawk2) [inline scripts/extract.arrow.shapes.pl]: Could you switch this to the form `grep length, map $_->as_text, @td`? It's slightly more efficient in not creating block contexts, as well as more concise.

File: scripts/extract.attributes.pl [hunk 2]
```diff
@@ -20,11 +20,10 @@
 	 S => 'subgraph',
 	);
 
-my(@content, @column);
-my(@row);
-my(@user);
+my(@column, @user);
 
-for my $c0 (map ref() ? [ $_->content_list ]->[0] : $_, map [ $_->content_list ]->[0], @td) {
+for my $c0 (map { $_->as_text() } @td) {
```
Comment (mohawk2) [inline scripts/extract.attributes.pl]: Same here.

**Review by mohawk2 — CHANGES_REQUESTED** [review]
Thank you for these! Just a couple of tweaks.

**Comment by nowthis** [comment]
I've made the requested change in new commit 341c6f6

**Review by mohawk2 — APPROVED** [review]

**Comment by mohawk2** [comment]
Released as 2.67. Thanks!

**Pull request merged**